ISeriesProvider/DB2iSeriesSqlBuilder.cs

```diff
@@ -457,41 +477,66 @@ protected override void BuildPredicate(ISqlPredicate predicate)
 
 					var ep = (SqlPredicate.ExprExpr)predicate;
 
-					if (ep.Expr1 is SqlFunction function && function.Name == "Date")
-					{
-						if (ep.Expr2 is SqlParameter parameter)
-						{
+                    //Temporary DateTime fix

```

Comment:
This is left from date/time fix I proposed a while back, I started from that codebase and I left it in, you can remove it if you want

Reply:
I'm happy for it to stay but would like unit tests to exercise it.

Reply:
To be honest, as this applies only to the Access Client driver and I've been using DB2Connect for some time now I can't remember the case clearly, but I'll try to find some time to address it